components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -6,18 +6,17 @@
     <oxd-button
       class="dropdown-btn"
       :class="dropdownButtonClasses"
-      :label="buttonData.label"
-      :iconName="buttonData.iconName"
+      :label="modelValue ? modelValue.label : buttonData.label"
+      :iconName="!modelValue ? buttonData.iconName : null"
```

Comment:
This logic is incorrect. What we are doing is if the modelValue is null set the iconName to null. It'll be correct for candidate list left panel but not generic for other places. That logic is not part of the SelectInputButton component and the icon to be visible or not should be decided by the candidate list and show/hide the icon. The component can provide a prop to control it.

Reply:
Fixed as per discussion in the review meeting on 2022-05-09
Now this validates using "doubleLineLabel" prop along with the modelValue